File: 3° Período/Estrutura de Dados I/Lista01/Codigo16/src/No.java
public class No {
    private int dado;
    private No proximo;

    public No(int nDado) {
        this.dado = nDado;
        this.proximo = null;
    }

    public int getDado() {
        return this.dado;
    }

    public No getProximo() {
        return this.proximo;
    }

    public void setDado(int nDado) {
        this.dado = nDado;
    }

    public void setProximo(No nProximo) {
        this.proximo = nProximo;
    }

    @Override
    public String toString() {
        return String.valueOf(this.dado);
    }
}

File: 3° Período/Estrutura de Dados I/Atividade12/model/No.js
class No {
  #dado;
  #proximo;

  constructor(nDado) {
    this.#dado = nDado;
    this.#proximo = null;
  }

  get dado() {
    return this.#dado;
  }

  get proximo() {
    return this.#proximo;
  }

  set dado(nDado) {
    this.#dado = nDado;
  }

  set proximo(nProximo) {
    this.#proximo = nProximo;
  }

  toString() {
    return this.#dado.toString();
  }
}
